resources/instance_sysv_init.rb

```diff
@@ -92,9 +92,6 @@
   def create_init
     include_recipe 'memcached::_package' unless new_resource.binary_path
 
-    # remove any runit instances with the same name if they exist
-    disable_legacy_runit_instance
```

Comment:
You're going to want to keep this for anyone upgrading from one of the older cookbook releases